tests/drivers/i2c/i2c_slave_api/testcase.yaml

```diff
@@ -1,8 +1,12 @@
+common:
+  depends_on: i2c
+  tags: drivers i2c
+  harness: ztest
+  harness_config:
+    fixture: i2c_bus_short
+
 tests:
-  drivers.i2c_slave:
-    depends_on: i2c
-    tags: drivers i2c
-    harness: ztest
-    harness_config:
-      fixture: i2c_bus_short
+  drivers.i2c_slave.stm32:
```

Comment:
Perhaps call this `drivers.i2c_slave.dual_role` in case it extends to other non-STM32 controllers?

Reply:
Done, thanks.